### ISSUE TITLE
Suppressing CA1824 in WPF project has no effect.

### ISSUE THREAD

**Issue by JohanLarsson** [dotnet/roslyn#49622]
**Steps to Reproduce**:

1. Create new WPF project
2. Add `<PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.1" PrivateAssets="all" />`
3. Add .editorconfig with dotnet_diagnostic.CA1824.severity = none
4. Build and see the warning still there.

https://github.com/JohanLarsson/SuppressWarningRepro

Maybe this is due to WPF projects copied to wpftmp during build?

**Expected Behavior**:

The warning to be suppressed.

**Actual Behavior**:

The warning still reported.

**Comment by Youssef1313** [comment]
Unlike most warnings, this specific warning is reported without a location:

```
CSC : warning CA1824: Mark assemblies with NeutralResourcesLanguageAttribute [D:\dotnet\SuppressWarningRepro\SuppressWarningRepro\SuppressWarningRepro.csproj]
```

i.e., it's reported on the csproj itself. The current editorconfig you have suppresses the warning for C# files. I've tried using `*.*` in editorconfig instead of `*.cs`, but the warning is still reported.

But still wanted to post in case this small piece of info is useful.

**Comment by Youssef1313** [comment]
cc: @mavasani

**Comment by mavasani** [comment]
I am unable to repro this after [migrating](https://docs.microsoft.com/visualstudio/code-quality/migrate-from-fxcop-analyzers-to-net-analyzers) from FxCopAnalyzers to .NET analyzers, and then enabling all rules as warnings by default:

```xml
<AnalysisLevel>latest</AnalysisLevel>
<AnalysisMode>AllEnabledByDefault</AnalysisMode>
```
